### ISSUE TITLE
Importing past flights with Airportcode TXL doesn't work

### ISSUE THREAD

**Issue by Maxklos** [johanohly/AirTrail#14]
### The bug

When I import past flight from Flightradar24 it works without error. But flights from and to the (now closed) Airport Berlin-Tegel TXL are "forgotten" and not displayed. Flight to the (also now closed) Airport Berlin Schönefeld SXF are converted to BER.
I can't manually add flight to/from TXL&SXF either, just BER. That's ok, but old data get lost this way.

Both versions could work:
- convert TXL flights to BER as well
- Allow import for TXL flights

Export file for testing (both TXL & SXF flights)
[flightdiary_2024_09_12_08_59.csv](https://github.com/user-attachments/files/16974926/flightdiary_2024_09_12_08_59.csv)

### Operating System that is running AirTrail

Ubuntu WSL

### AirTrail Version

latest

### Reproduction steps

1.import file with airport code TXL
2.flights won't appear

**Comment by Maxklos** [comment]
Yes, in the CSV this flight is SXF -> ARN but in Airtrail it's BER -> ARN. 

![image](https://github.com/user-attachments/assets/af52d046-07ca-4ff6-8f69-c0043353c5ba)
![image](https://github.com/user-attachments/assets/4f35e188-01ed-4cd1-82ed-5ec2d1a0a26a)

All my flights pre 2020 (closing of SXF/opening BER) got converted to BER

**Comment by johanohly** [comment]
The airports are matched by their ICAO code, rather than their IATA code, to support airports without a IATA code. In AirTrails' database, `Berlin Brandenburg Airport`s ICAO code is `EDDB` which, in your CSV, matches `Berlin Schoenefeld`. I think the ICAO code got transferred when Schoenefeld shut down.

**Comment by johanohly** [comment]
Handled in v0.1.0, you would have to re-import the flights though

**Comment by Maxklos** [comment]
reimported my flight, everything works as expected. Thank you very much!
